toolium/behave/env_utils.py

```diff
@@ -216,9 +216,9 @@ def __execute_steps_by_action(self, context, action):
                     context.execute_steps(u'''%s%s''' % (GIVEN_PREFIX, self.__remove_prefix(item)))
                     self.logger.debug(u'step defined in pre-actions: %s' % repr(item))
                 except Exception as exc:
-                    if action in [ACTIONS_BEFORE_FEATURE]:
+                    if action in [ACTIONS_BEFORE_FEATURE] or action in [ACTIONS_AFTER_FEATURE]:
```

Comment:
unify in one comparison
`if action in [ACTIONS_BEFORE_FEATURE, ACTIONS_AFTER_FEATURE]:`

Reply:
Done